.github/workflows/build.yml

```diff
@@ -37,16 +37,12 @@ jobs:
           path: |
               data
               jars
```

Comment:
Perhaps we should remove `jars` from being cached?

As I understand it, Gradle will run the `copyJars` task as part of `compileJava`.
And gradle is also cached by the `actions/setup-java` step.

Reply:
We could remove it, technically, Gradle will be looking elsewhere for the jars anyways.   For me, it puts the dependencies in a local home directory called `~/.gradle/caches`.  The `jars` directory will be for users outside of Gradle and for legacy purposes.

Reply:
Sorry I wasn't clear. I meant to remove it from being cached redundantly for CI, since it should be re-populated by Gradle (from its cache or not) anyway. If we want to speed it up even more we could stop `copyJars` from being compulsory, for users (like CI) who run everything only through Gradle, but that can be a topic for another day.